client/layout/guided-tours/config-elements.js

```diff
@@ -6,35 +8,52 @@ import classNames from 'classnames';
 import { translate } from 'i18n-calypso';
 import {
 	debounce,
+	defer,
 	find,
 	mapValues,
 	omit,
 	property,
+	startsWith,
 } from 'lodash';
+import debugFactory from 'debug';
```

Comment:
I talked to a few colleagues and the consensus seems to be to keep the more useful `debug` statements. I went through those we still have in here and think that all of them would be useful beyond the issue addressed in this PR — they all help us understand the life cycle of a tour. That will be helpful in the future. 

Also in production, these all become no-ops: https://github.com/Automattic/wp-calypso/blob/master/webpack.config.js#L147

TL;DR: yes.